src/main/webapp/resources/js/components/samples/components/GenomeAssemblyList.tsx

```diff
@@ -1,34 +1,55 @@
 import React from "react";
 import { Button, notification, Popconfirm, Tag, Tooltip } from "antd";
-import { useDispatch, useSelector } from "react-redux";
+import { RootStateOrAny, useDispatch, useSelector } from "react-redux";
 
 import { SequenceFileTypeRenderer } from "./SequenceFileTypeRenderer";
 import { downloadGenomeAssemblyFile } from "../../../apis/samples/samples";
 import { GenomeAssemblyListItem } from "../../sequence-files/GenomeAssemblyListItem";
 import { DEFAULT_ACTION_WIDTH } from "../sampleFilesSlice";
 import { useUpdateDefaultSampleGenomeAssemblyMutation } from "../../../apis/samples/samples";
 import { setDefaultGenomeAssembly } from "../sampleSlice";
+import { SampleGenomeAssembly } from "../../../types/irida";
+
+export interface GenomeAssemblyListProps {
+  removeSampleFiles: ({
+    fileObjectId,
+    type,
+  }: {
+    fileObjectId: string | number;
```

Comment:
Should this be both?

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f